app/src/components/widgets/charts/cumulativeTrendChart/chartjsConfig.js

```diff
@@ -15,43 +14,23 @@ const getExecutions = () => [
   'statistics$executions$passed',
 ];
 
-// const messages = {
-//   statistics$executions$failed: commonMessages.failed,
-//   statistics$executions$skipped: commonMessages.skipped,
-//   statistics$executions$passed: commonMessages.passed,
-//   statistics$defects$product_bug$total: commonMessages.pb,
-//   statistics$defects$automation_bug$total: commonMessages.ab,
-//   statistics$defects$system_issue$total: commonMessages.si,
-//   statistics$defects$no_defect$total: commonMessages.nd,
-//   statistics$defects$to_investigate$total: commonMessages.ti,
-// };
+const getTotal = () => ['statistics$executions$total'];
 
-const convertIntoPercents = (columns) => {
-  const percentageColumns = [];
+const convertIntoPercents = (datasets) => {
+  const totalDataset = Object.assign({}, datasets[0]);
 
-  columns.forEach((column) => {
-    const newColumn = [];
-
-    newColumn.push(column[0]);
-
-    for (let i = 1; i < column.length; i += 1) {
-      const total = columns.reduce((acc, item) => acc + item[i], 0);
-
-      newColumn[i] = -(-column[i] / total * 100).toFixed(2);
-    }
+  return datasets.map((dataset) => {
+    Object.assign(dataset, {
```

Comment:
Mutating arguments is not a good idea.
`Object.asign({}, dataset, { .. })` or `{...dataset, ...{ .. })`

Reply:
fixed